src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -128,7 +136,7 @@ Loading configuration from config file ./config.toml
 
 </CodeBlock>
 
-By default, only the API is enabled, if you want to enable the HTTP or UDP trackers, you need to change the `enabled` value in the `./config.toml` file.
+<!-- By default, only the API is enabled, if you want to enable the HTTP or UDP trackers, you need to change the `enabled` value in the `./config.toml` file.
```

Comment:
HI @ivan9989-pkj why is this comment hidden? We can show it. However, it's wrong. For the development default config now all services are enabled by default. See the tracker.development.sqlite3.toml file.